mycore-ifs/src/main/java/org/mycore/datamodel/niofs/ifs2/MCRFileChannel.java

```diff
@@ -32,22 +32,32 @@
 import org.mycore.common.content.streams.MCRMD5InputStream;
 import org.mycore.datamodel.ifs.MCRContentInputStream;
 import org.mycore.datamodel.ifs2.MCRFile;
+import org.mycore.datamodel.niofs.MCRFileAttributes;
+import org.mycore.datamodel.niofs.MCRPath;
 
 /**
  * @author Thomas Scheffler (yagee)
  */
 public class MCRFileChannel extends FileChannel {
 
+    private final MCRPath path;
+
     private FileChannel baseChannel;
 
     private MCRFile file;
 
-    private boolean write;
+    private boolean write, modified, create;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5215111)